src/lib/tvgRender.h

```diff
@@ -119,6 +119,7 @@ struct RenderShape
     RenderStroke *stroke = nullptr;
     uint8_t color[4] = {0, 0, 0, 0};    //r, g, b, a
     FillRule rule = FillRule::Winding;
+    bool fillUnder = true;
```

Comment:
I'd like put this in the stroke structure since it's stroking dependency feature so far. it could save the memory.

Reply:
but then:
```
    auto shape = tvg::Shape::gen();
    shape->appendRect(0, 0, 200, 200, 0, 0);
    shape->fill(255, 255, 0, 255);
    shape->fill(true);       //useless, but user can call it
```
user called only the fill apis, no stroke will be drawn, fill(true) is useless.
but because of fill(true) is called I have to create a stroke just to put there the information, that won't be used.

if the api is shape->stroke(true/false) then it's ok to create a stroke - a user should be aware, that calling some 'stroke' api will create a stroke. but calling a fill api that creates a stroke is strange...

can we introduce a different name, like shape->order(bool strokeUnder) ? then it's not so confusing.

Reply:
@mgrudzinska 

shape->order(true) //useless, but user can call it  
shape->fill(true) //userless but user can call it.
shape->stroke(true) //useless but user can call it.

Nothing different.

Only a few thoughtless programmers would call fill(true) without using stroke. It's best to ignore such cases as they are not critical and cannot be covered.

Reply:
@mgrudzinska let's go with this:

Result order(bool strokeFirst) noexcept;        //drawing order